TESTS/netsocket/udp/udpsocket_echotest.cpp

```diff
@@ -41,11 +40,20 @@ static const int BUFF_SIZE = 1200;
 char rx_buffer[BUFF_SIZE] = {0};
 char tx_buffer[BUFF_SIZE] = {0};
 
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE != MESH
 static const int PKTS = 22;
 static const int pkt_sizes[PKTS] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, \
                                     100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, \
                                     1100, 1200
                                    };
+static const double TOLERATED_LOSS_RATIO = 0.3;
+#else
+static const int PKTS = 13;
+static const int pkt_sizes[PKTS] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, \
+                                    100, 200, 300
+                                   };
```

Comment:
Then who tests that those bigger packets can pass if we are not going to?

Reply:
In most cases more than 300 bytes packet fails.

Reply:
Im going to setup simple local mesh network in our office to verify if those tests will pass under no clear RF condition

Reply:
Test skip reverted. It is successful under  local mesh BR. Testing mesh performance under heavy RF traffic is beyond the scope of netsockets tests.